.github/workflows/ci.yml

```diff
@@ -263,6 +359,10 @@ jobs:
         . /opt/intel/mkl/bin/mklvars.sh
         export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/${{ matrix.build-type }}
         export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/${{ matrix.build-type }}/libs
+        if [ ${{ matrix.compiler }} = gcc ]; then
+          export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/external_libraries/mmg/mmg_5_5_1/lib
+        elif [ ${{ matrix.compiler }} = clang ]; then
+          export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/external_libraries/mmg/mmg_5_4_1/lib
```

Comment:
hm this grows more and more out of hand...
did you try adding both to the path (in this PR?)

I will think of a better solution for the future, sth like an env file for the docker

Reply:
Yep, not a fan of this solution either.

Let's check if it works fine like this and then I will try adding both.